### PR TITLE
feat: implemented changes as requested in #77

### PR DESCRIPTION
**Pull request by M4dhav**
Added Spacing as needed, and Title Tile for Settings as needed #77

### CODE REVIEW

File: lib/presentation/organisms/timer_organism.dart [hunk 1]
```diff
@@ -100,29 +100,43 @@ class _TimerOrganismState extends State<TimerOrganism>
...
-          onIcon: Icons.lock_open,
-          onChange: (bool value) {
-            lockScreen = value;
-            WakeLockController.instance.setWakeLock(lockScreen);
-          },
-          initialValue: lockScreen,
-        ),
-      ],
+    final Widget body = SizedBox(
+      width: 200,
```
Comment (guyluz11) [inline lib/presentation/organisms/timer_organism.dart:104]: If we can set it with max width but not minimum like it behaved before that will be nice.

File: lib/presentation/organisms/timer_organism.dart [hunk 2]
```diff
@@ -100,29 +100,43 @@ class _TimerOrganismState extends State<TimerOrganism>
...
+        children: [
+          TopBarMolecule(
+            title: "settings",
+            topBarType: TopBarType.none,
+          ),
+          SeparatorAtom(
+            variant: SeparatorVariant.generalSpacing,
+          ),
+          ToggleButtonMolecule(
+            text: 'Sound',
```
Comment (guyluz11) [inline lib/presentation/organisms/timer_organism.dart:116]: Please add translation

File: lib/presentation/organisms/timer_organism.dart [hunk 3]
```diff
@@ -100,29 +100,43 @@ class _TimerOrganismState extends State<TimerOrganism>
...
+            offIcon: Icons.music_off_rounded,
+            onIcon: Icons.music_note_rounded,
+            onChange: (bool value) =>
+                PlayerController.instance.setSilentState(!value),
+            initialValue: !PlayerController.instance.isSilent(),
+          ),
+          SeparatorAtom(),
+          ToggleButtonMolecule(
+            text: 'Screen Lock',
```
Comment (guyluz11) [inline lib/presentation/organisms/timer_organism.dart:125]: Please add translations

**Review by guyluz11 — CHANGES_REQUESTED** [review]
If you can also change the off toggle button colors to something else like gray (outline) from the theme it would be nice :)

**Comment by M4dhav** [comment]
this is how it looks now:
image
I believe we should add some kind of bounding for the buttons to make it look better, I can either do that with padding or make it responsive using MediaQueries.
Also, I have noticed that the app is not currently responsive at all and we are frequently using absolute integers to set sizes of elements. Do you want to make these responsive in the near future?

File: lib/presentation/atoms/toggle_button_atom.dart [hunk 4]
```diff
@@ -29,7 +29,14 @@ class _ToggleButtonAtomState extends State<ToggleButtonAtom> {
 
   @override
   Widget build(BuildContext context) {
+    final ThemeData themeData = Theme.of(context);
+    final ColorScheme colorScheme = themeData.colorScheme;
     return AnimatedToggleSwitch<bool>.rolling(
+      style: ToggleStyle(
+        backgroundColor: colorScheme.outline,
```
Comment (guyluz11) [inline lib/presentation/atoms/toggle_button_atom.dart:36]: Please change the background color to outline only when the toggle is off.

Maybe also the borderColor, test how does it look

File: lib/presentation/atoms/toggle_button_atom.dart [hunk 5]
```diff
@@ -29,7 +29,14 @@ class _ToggleButtonAtomState extends State<ToggleButtonAtom> {
...
+        borderColor: colorScheme.outline,
+        indicatorColor: colorScheme.primary,
```
Comment (guyluz11) [inline lib/presentation/atoms/toggle_button_atom.dart:38]: Maybe secondary or tertiary color for off-state will look better.

**Review by guyluz11 — CHANGES_REQUESTED** [review]

**Comment by guyluz11** [comment]
Regarding the response thing the widgets should not look proportionally the same for all screen sizes.
And currently our main focus is mobile, new features, bug fixes.
So responsive for big screen sizes can wait for now.

Also I didn't find the use of the scale widget in the atoms folder so maybe you meant something else.

**Comment by M4dhav** [comment]
> Regarding the response thing the widgets should not look proportionally the same for all screen sizes.

Well I mean obviously we don't want them to look the same on a Laptop sized screen for Windows or a Web build but for phones we should still want the apps to look similar I think
> Also I didn't find the use of the scale widget in the atoms folder so maybe you meant something else.
What I meant was that the widgets aren't scaling with the screen, as in they aren't getting bigger or smaller or maintaining their aspect ratio to the screen

**Comment by M4dhav** [comment]
image
For the color changes, is this good?

**Comment by guyluz11** [comment]
The last one but remove the border-color

**Comment by guyluz11** [comment]
Hy, is this pr ready?, it looks good but you didn't press Re-requested review button 

image

**Comment by M4dhav** [comment]
yes, the PR is ready

**Review by guyluz11 — APPROVED** [review]

**Comment by guyluz11** [comment]
Can you comment on the issue

I have opened a pr

**Pull request merged**